Recording/SendItemInfo.s

```diff
@@ -122,6 +123,20 @@ SendItemInfo_AddToBuffer:
   stb r3,OFST_METADATA_3(REG_Buffer)
   lbz r3,0xDEF(REG_ItemData) # This stores charge power for Samus/MewTwo (0-7)
   stb r3,OFST_METADATA_4(REG_Buffer)
+# Store item ownership
+  lwz r3, 0x518(REG_ItemData)
+  cmpwi r3, 0x0   # Is this a null pointer?
+  beq DontFollowItemOwnerPtr
+  bne FollowItemOwnerPtr
```

Comment:
This line is unnecessary, even if it takes the branch it goes to the next line - same result as if it doesn't take the branch